.gitattributes

```diff
@@ -1,2 +1,2 @@
-# Declare files that will always have LF line endings on checkout.
+# Declare that the script files will always have LF line endings on checkout. Otherwise using cygwin and WSL to execute these will fail.
```

Comment:
```suggestion
# Declare that the script files will always have LF line endings on checkout.
# Otherwise using cygwin and WSL to execute these will fail if checkout is done on Windows..
```

Reply:
Not sure how to fix that myself using the web interface, but the second line has one period too many at the end.

Reply:
Fixed